00-Install_and_Setup/README.md

```diff
@@ -121,6 +121,28 @@ out-of-date, always update to its pre-release version with pip:
 
     (astropy-workshop) % pip install astroquery --pre --upgrade
 
+## 5. Check Jupyter Notebook
+
+In the conda environment created above, go into the directory
+of one of the chapters with notebooks (files ending with `.ipynb`)
+and start Jupyter notebook:
+
+    (astropy-workshop) % jupyter notebook
```

Comment:
I believe this will now be the first time a participant is instructed to use `jupyter notebook`. I have been thinking we'd use Jupyterlab, but I see in the intro materials, which we won't cover in the workshop now, that we show `jupyter notebook`. I'll make a separate issue for whether we switch the instructions to Jupyterlab.